packages/storybook-utils/src/mandatory.ts

```diff
@@ -0,0 +1,44 @@
+import { type Decorator } from "@storybook/vue3";
+import { ref, watch } from "vue";
+
+type MandatoryIndicator = "required" | "optional";
```

Comment:
We agreed to call it "required" instead of mandatory :)

Reply:
Also for the file name

Reply:
done